src/Components/MessagePost.js

```diff
@@ -63,12 +67,19 @@ class MessagePost extends Component {
                             <p className="message-post-message">{this.props.content}</p>
                         </MDBCol>
                     </MDBRow>
+
+                    <MDBRow>
+                        <MDBCol size="12">
+                            <MDBIcon 
+                                icon="reply"
+                                onClick={this.handleShowCommentsClick} 
+                                className="grey-text message-show-comments-btn"
```

Comment:
Just a thought can you put the className & onClick on the row and not the icon? So the hit box is bigger? Esp when we add in the number of replies text it will make a nicer hit box for users similar to slack.

Reply:
Sounds good